source/documentation/joiners_leavers.md

```diff
@@ -0,0 +1,73 @@
+# Joiners & Leavers
+
+This should be a complete list of tools/services that new technology team
+joiners should be given access to, and leavers be removed from.
+
+
+## Joiners checklist
+
+### Try it out!
+
+1. [Sign up to GovWifi!](https://www.wifi.service.gov.uk/about-govwifi/connect-to-govwifi/)
+2. Ask your tech lead to get added to the [Admin Portal](https://admin.wifi.service.gov.uk/) and [Admin Portal - Staging](https://admin.staging.wifi.service.gov.uk/)
+
+### Infrastructure
+
+1. [Request an AWS account](https://gds-request-an-aws-account.cloudapps.digital/).
+2. Ask RE:D to add you to GovWifi account(s).
+3. [Set up your access to the infrastructure](/accessing-the-infrastructure.html) - includes access to databases and ssh.
+4. We use GovPaaS in some places. See [Gov.uk Platform as a Service - Get started](https://docs.cloud.service.gov.uk/get_started.html). Once you have an account, ask your Tech Lead to add you to the `govwifi` organization.
+
+### Secrets
+
+See [secrets](/secrets.html) for getting access to shared secrets, certificates and keys. Someone on your team will need to add you.
+
+### Development and deployment
+
+- Ask your Tech Lead to add you to the [alphagov organisation](https://github.com/alphagov) and [GovWifi team](https://github.com/orgs/alphagov/teams/govwifi).
+- The main GovWifi repositories are listed under [the application overview](/index.html). Just follow the instructions on the repository.
+- We use [Concourse](https://cd.gds-reliability.engineering/) for deployment - it uses GitHub OAuth. See [Deploying](/deploying.html) to learn how we use it.
+
+To understand how all fits together, [check the Service Operation Manual](https://docs.google.com/document/d/1XvzfnjQf2kCbArJBiI6B6_pnvZOCkwX-N1WyTPAZfWI/edit).
+
+### Monitoring and support
+
+- Alerts and logs can be found on [Cloudwatch](https://eu-west-2.console.aws.amazon.com/cloudwatch/home?region=eu-west-2#).
+- We have SMS set up. **TODO: Emile, please fill in!!**
+- [Sentry SaaS](https://sentry.io/organizations/government-digital-services/projects/) uses a shared password, see [secrets](secrets).
```

Comment:
docs are not set up to accept these kind of hrefs, so requires `.html` on the end.

Suggest setting up link references at the end of the file:

```md
see [secrets][secrets]

...

[secrets]: /secrets.html
```

Reply:
Worked locally, but I'll convert everything just in case

Reply:
worked locally but making suggested changes anyway